README.md

```diff
@@ -117,6 +117,10 @@ The following options are available (defaults are shown below):
     titleBoost: 5,
     contentBoost: 1,
     parentCategoriesBoost: 2, // Only used when indexDocSidebarParentCategories > 0
+    // Add urls to be excluded from the search index. All versions of the given url will be ignores.
+    // If a baseUrl is set in docusaurus.config.js it will be prepended
+    ignore: ['/docs/reference/changelog']
+
```

Comment:
```suggestion
    // Add pages to be excluded from the search index. A given page will be ignored for all
    // documentation versions. If a baseUrl is set in docusaurus.config.js, it will be prepended.
    // Similarly, language prefixes will be prepended.
    ignore: [
      // e.g.: '/docs/reference/changelog'
    ]
```

I have some minor suggestions for rewording this slightly, and I would prefer if the example url is only added as part of a comment, since this structure should reflect the default options ("The following options are available (defaults are shown below):").

Reply:
Also, could you move the option out of the `lunr` object into the `options` object? I feel like it should be located right after the `indexDocs`, `indexBlog`, and `indexPages` options.